.github/workflows/test.yaml

```diff
@@ -51,9 +51,17 @@ jobs:
           echo "SDK Branch: android-sdk@${SDK_BRANCH}"
           echo "API Level: ${{ matrix.api-level }}"
 
+      - name: Check out Common Java SDK
```

Comment:
temporary until java.v4 is published (which we'll do when this and the java-server PRs are ready for merge)